### DISCUSSION TITLE
Lego Support Google Public CA?

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [go-acme/lego#1696] [Q&A]
Hi google released beta Public CA Around march maybe... i used acme.sh until now! I wanted to try lego but when i try to register google public ca it fails... lego cannot register google public ca! 
```
2022/08/12 20:02:51 [INFO] acme: Registering account for hello@example.com
2022/08/12 20:02:54 Could not complete registration
        acme: error: 0 :: POST :: https://dv.acme-v02.api.pki.goog/new-account :: urn:ietf:params:acme:error:unauthorized :: External Account Binding JWS verification failed
```
I think its not supported but still want to be sure!...

**Comment by ldez** [comment]
Hi,

from what I can read Google follows the RFC8555 so it must work.

https://cloud.google.com/blog/products/identity-security/automate-public-certificate-lifecycle-management-via--acme-client-api

**Reply by IzumiSenaSora** [reply]
```
2022/08/12 20:02:51 [INFO] acme: Registering account for hello@example.com
2022/08/12 20:02:54 Could not complete registration
        acme: error: 0 :: POST :: https://dv.acme-v02.api.pki.goog/new-account :: urn:ietf:params:acme:error:unauthorized :: External Account Binding JWS verification failed
```

I am getting this error!... i followed same instruction! In acme.sh its working fine! I tried with letsencrypt via lego its ok too!
